fabfile.py

```diff
@@ -756,19 +732,77 @@ def update_npm_requirements(force_reinstall=False):
         else:
             venvcmd('npm update', chdir=False)
 
-def _install_builddeps():
+
+@task
+def install_single_server():
+    """
+    Will install all assembl components on a single server
+    """
+    sanitize_env()
+    execute(skeleton_env, None)
+    execute(install_database)
+    execute(install_assembl_server_deps)
+
+
+@task
+def install_assembl_server_deps():
+    """
+    Will install most assembl components on a single server, except db
+    """
+    sanitize_env()
```

Comment:
load_environment gives us strings; env variables declared in the fabfile env_... can be strings, lists or bools. sanitize_env ensures type uniformity between both origins.